packages/client/src/routes.test.ts

```diff
@@ -1,17 +1,13 @@
 import slugify from "slugify";
 
-import { getslug, geturl } from "./routes";
+import { geturl } from "./routes";
 
 const text =
   "U gaat het dakraam plaatsen aan de achterkant van het gebouw. Ligt die kant aan een straat, fietspad of voetpad waar iedereen mag komen?";
 const slug =
```

Comment:
We don't use question slugs anymore right? Where is this slug being used.

Reply:
This slug is just a test to see if the conversion works correctly in `slugify`